gitbook2pdf/libs/gitbook.css

```
@@ -957,3 +957,12 @@ Orginal Style from ethanschoonover.com/solarized (c) Jeremy Hull <sourdrums@gmai
 .book.color-theme-2 .book-body .page-wrapper .page-inner section.normal code .xml .hljs-cdata {
   opacity: 0.5;
 }
+
+/* Make each chapter start on a new page in the PDF output */
+section.normal + section.normal,
+section.normal + h1,
+h1 + section.normal,
+h1 + h1 {
+  page-break-before: always;
```

Stylelint (17.9.0)

[error] Expected "page-break-before" to be "break-before" (property-no-deprecated)

(property-no-deprecated)